src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -623,7 +623,8 @@ bool primitive_inst::need_reset_output_memory() const {
             auto feature_block_size = get_feature_block_size(fmt);
```

Comment:
random spot) could you add a test case? I think you can add some in `mem_reset_test.cpp`
Please write PR description as well.